addons/openebs/enable

```diff
@@ -30,12 +31,17 @@ fi
 # make sure the "openebs" namespace exist
 $KUBECTL create namespace "$OPENEBS_NS" > /dev/null 2>&1 || true
 
+# adding check for small clusters by checking number of nodes in cluster
+K8S_NODES=$( ${KUBECTL} get nodes | awk '!/NAME/{count++} END{if(count > "2"){print 3} else {print count}}' )
+echo "Setting Jiva Replicas to ${K8S_NODES}"
+
```

Comment:
I would not like this change, because it may break workflows where openebs is enabled but more nodes are added afterwards. In this case, having 3 jiva replicas would work, and it is a breaking change that we would like to avoid.

Can you please change this to instead be a simple configuration option? Or even print a help message after enabling the addon, with instructions on how to change the number of replicas (e.g. using a `helm upgrade ...` command).

Reply:
That sounds good, I'll get that changed made and update the pr

Reply:
I have updated the pr to keep the default behavior of having 3 jiva replicas with the ability to use the -r flag to override that value with the following command `microk8s enable openebs -j 1` which should help out with developers running small development clusters. This also adds the ability to update the chart version and bumps that version to 3.3.x to fix the problem with k8s 1.25